test/module-enabled/main.tf

```
@@ -0,0 +1,15 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# UNIT TEST MODULE
+# This module exists for the purpose of testing only and should not be
+# considered as an example.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+provider "aws" {
+  region = var.aws_region
+}
+
+module "test" {
+  source = "../.."
+
+  module_enabled = var.module_enabled
```

this will always test disabled only no need to parameterize.
```suggestion
  module_enabled = false
```

---

resolved in 82eb622